docs/reference/ilm/actions/ilm-searchable-snapshot.asciidoc

```diff
@@ -30,6 +30,14 @@ To keep the snapshot, set `delete_searchable_snapshot` to `false` in the delete
 Specifies where to store the snapshot. 
 See <<snapshots-register-repository>> for more information.
 
+`force_merge_index`::
+(Optional, boolean)
+Force merges the managed index to one segment.
+Defaults to `true`.
+If the managed index was already force merged using the
+<<ilm-forcemerge, force merge action>> in the `cold` or `warm` phase
```

Comment:
Super minor, but we allow force merge in "hot", so maybe we should just say 

```
If the managed index was already force merged using the
<<ilm-forcemerge, force merge action>> in a previous action
the `searchable snapshot` action force merge step will be a no-op.
```

Reply:
We might also want to mention that force merging is best effort, since it's possible for it to fail (and we still continue)